junifer/configs/juseless/datagrabbers/ixi_vbm.py

```diff
@@ -0,0 +1,91 @@
+"""Provide class for juseless datalad datagrabber."""
```

Comment:
Maybe mention IXI like so: `"""Provide class for IXI VBM juseless datagrabber."""`

Reply:
yes indeed